### PR TITLE
Feature/conversions

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:**
introduce conversion metrics

**This PR will result in the following new package version:**

Breaking change. Adding new fields 

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

## Features: Conversion Metrics
- Introduces the following conversion fields to the Reddit Ads `<>_report` models (source fields in parenthesis):
  - `conversions` (click_through_conversion_attribution_window_month)
  - `view_through_conversions` (view_through_conversion_attribution_window_month)
  - `total_items` (total_items)
  - `total_value` (total_value)
- Introduces the variable `<>_conversions_passthrough_metrics` to allow additional fields from the source `_conversion_report` tables.
- We use the maximum attribution window when considering conversions and therefore retrieve conversions metrics from the `click_through_conversion_attribution_window_month` (conversions) and `view_through_conversion_attribution_window_month` (view_through_conversions) fields in the respective source tables. You may bring in additional windows and fields through via the `<>_conversions_passthrough_metrics` variable. For information on how to configure these variables, refer to the [README](https://github.com/fivetran/dbt_reddit_ads/blob/main/README.md#passing-through-additional-metrics).

## Under the hood
- Added the respective seed data for the new models in addition to updating relevant documentation.
- Added documentation explaining potential discrepancies across reporting grains.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [na] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [na] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

image

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

**Comment by fivetran-reneeli** [comment]
will close out issue #3 as well since this is addressed

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 packages:
-- package: fivetran/reddit_ads_source
-  version: [">=0.2.0", "<0.3.0"]
+# - package: fivetran/reddit_ads_source
+#   version: [">=0.3.0", "<0.4.0"]
+
+  - git: https://github.com/fivetran/dbt_reddit_ads_source.git
```
Comment (fivetran-jamie) [inline packages.yml:5]: switch

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
+# dbt_reddit_ads v0.3.0
+[PR #13](https://github.com/fivetran/dbt_reddit_ads/pull/13) includes the following updates:
+
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:3]: We don't mention the upstream new staging models here at all or reference the other breaking changes in the source package. Can we make sure to include those here.

File: integration_tests/tests/consistency/consistency_account_report.sql
Comment (fivetran-joemarkiewicz) [inline integration_tests/tests/consistency/consistency_account_report.sql:1]: Request to add the conversion metrics to the validation tests but comment them out so they may be added back within a future release and we don't forget about them.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-jamie @fivetran-reneeli great job on this PR! @fivetran-jamie there are a few small changes I would like to be addressed before this is ready for release.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

**Pull request merged**